mmf/modules/detr/datasets/transforms.py

```diff
@@ -0,0 +1,306 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
```

Comment:
Are none of the functions in this file available in torchvision transforms?

Reply:
These transforms (now moved to `mmf/utils/image_transforms.py`) are not available in torchvision transforms. The torchvision transforms only handle images, while the transforms here handle both images and the object detection targets (boxes).

Also, I kept `DETRImageProcessor` as a new processor since its corresponding transforms are not available in torchvision transforms.